internal/util/random.go

```diff
@@ -46,3 +46,17 @@ func RandomViewerCount() int64 {
 	}
 	return viewer.Int64()
 }
+
+// RandomViewerCount generates a fake viewercount between 0->10,000,000
+func RandomType() string {
+	someInt, err := rand.Int(rand.Reader, big.NewInt(1*10*100*100*100))
+	if err != nil {
+		log.Fatal(err.Error())
+	}
+	if (someInt.Int64()%2) == 0{
+		return "bits"
+	}else
+	{
+		return "subscription"
+	}
+}
```

Comment:
Is this just to give the contributions as random contributions? Since it's not really a generally resuable component, you could do this using Random viewer or just hardcode a contribution within the Hype Train event code.